main.go

```diff
@@ -142,13 +169,16 @@ func run(ctx context.Context, onStart *func(), command string, args ...string) e
 	select {
 	case <-ctx.Done():
 		return nil
-	case <-time.After(debounceTimeout):
+	case <-time.After(time.Duration(config.Test().Debounce) * time.Millisecond):
```

Comment:
Debounce should be per Rule :thinking: 
You can skip this for now. I will open another issue in the future.

Reply:
I think I got this